projects/systelab-login/src/lib/change-password-dialog.component.spec.ts

```diff
@@ -15,4 +15,10 @@ describe('ChangePasswordDialogParameter', () => {
 
         expect(parameters.hasNumpad).toBeFalsy();
     });
+
+    it('should set userName by default in change password dialog', () => {
+        const parameters = new ChangePasswordDialogParametersMock();
+
+        expect(parameters.userName.length).toBeGreaterThan(0);
```

Comment:
This expectation is just asserting on the userName field not empty. It is not asserting on what is suggested by it's description (that the specific default value is displayed).

What about asserting on it being equal than the real expected value that is expected?